module/__init__.py

```diff
@@ -739,7 +765,38 @@ def __init__(self, unpacker):
 
 
 class Event(Response):
-    pass
+
+    def __init__(self, unpacker):
+        # This is here for debugging purposes!
+        self.unpacker = unpacker
+
+        Response.__init__(self, unpacker)
+
+        # If this is a xcb_ge_generic_event_t (response type 35) then we need a few more fields
+        if self.xge and isinstance(unpacker, CffiUnpacker):
+            self.extension, self.length, self.event_type, self.full_sequence = unpacker.unpack("xB2xIH22xI")
+
+            # There's some extra work to do if the event has data past the 32 byte boundary
+            if self.length:
+
+                # Calculate the size of the original buffer. This is 4 bytes short as it seems to omit the `full_sequence` field
+                buffer_size = 32 + (self.length * 4) + 4
+                
+                # Create a buffer object that's the full size
+                buffer = ffi.buffer(unpacker.cdata, buffer_size)
+
+                # Copy the event to the new buffer and skip the `full_sequence` field
+                # newbuffer[0:32] = buffer[0:32]
```

Comment:
Unused?

Reply:
No. I manipulate that buffer in the next line and pass it to the unpacker a few lines after that. Or is your point that we shouldn't need to create a buffer at all?

Reply:
I guess I just don't understand what this commented out code is supposed to mean. Is it unused code from when you were testing? Or is it intended to illustrate something?

Reply:
Sorry. Misunderstood you. Yes, it's unused code from before. Have deleted.